markdown_flow/providers/openai.py

```
@@ -0,0 +1,369 @@
+                "processing_time": processing_time_ms,
+                "timestamp": int(time.time()),
+                "stream_done": True,
+            }
+            self._last_metadata = metadata
+
+            # Debug output: Stream completion info
+            if self.config.debug:
+                self._print_response_metadata(metadata)
+
+        except Exception as e:
+            raise ValueError(f"Streaming request failed: {str(e)}") from e
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)